code/modules/customitems/item_defines.dm

```diff
@@ -748,6 +748,18 @@
 	desc = "A worn leather jacket. Some burn holes have been patched."
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	icon_state = "dusty_jacket"
+	
+/obj/item/clothing/suit/fluff/supplymaster_jacket //Denthamos: Henry Grandpa Gadow
+	name = "faded NT Supply Master's Coat"
+	desc = "A faded leather overcoat bearing a worn out badge from the NAS Crescent on the shoulder, and a designation tag of Supply master on the front.  A tarnished gold nameplate says H.Gadow on it."
```

Comment:
`Supply master` -> should be `Supply Master`

But also has the potential for confusing newbies into thinking they're the QM - which it shouldn't.

Reply:
Righty, I’ll fix the grammar as soon as I get home, as for the second part you are probably better off asking Denthamos. I’m just the code monkey